Cargo.toml

```diff
@@ -80,12 +82,13 @@ vergen = { version = "7", default-features = false, features = [
 ] }
 
 [features]
-default = []
+default = ["generate-completions"]
```

Comment:
I added completions here just to make sure the less conventional features like build's `--up` option aren't breaking it.

I'll make sure to remove once this is ready for review again.

Reply:
Ooh, if this provides us with a better and more consistent way to https://github.com/fermyon/spin/pull/1032 then I'd love to have your feedback on that PR (or your own PR).  Thanks!  (I agree on not trying to shoehorn it into this one, of course.)